### PR TITLE
Updated mobian wiki link

### PR DESCRIPTION
**Pull request by imadnyc**
Updated mobian wiki link

### CODE REVIEW

File: usage/install-on-desktops.md [hunk 1]
```diff
@@ -14,9 +14,9 @@ Thanks to the Postmarket community, you can also find instructions and troublesh
 
 ## Mobian
 
-The community was kind enough to post the install instructions for Mobian on their Doku pages:
+The community was kind enough to post the install instructions for Mobian on wiki:
 
-{% embed url="https://wiki.mobian-project.org/doku.php?id=waydroid" %}
+{% embed url=""https://wiki.debian.org/Mobian/How-to#How_to_run_Android_apps_via_Waydroid %}
```
Comment (aleasto) [inline usage/install-on-desktops.md:19]: the double quotes are misplaced

**Review by aleasto — COMMENTED** [review]
I'm not quite happy with how old the provided packages are (more than 1 year)

I don't know Mobian well enough, but shouldn't they just use the same packages we build for Debian? That's what the current guide says anyways

**Comment by imadnyc** [comment]
Waydroid doesn't currently support trixie, the current distro that Mobian is based on. Maybe I'll add support later, but for now I _think_ this is the only way to install it.

**Comment by aleasto** [comment]
We thought we already supported trixie. Turns out we forgot to flip the switch to enable it in the install script. That's now fixed.

**Comment by imadnyc** [comment]
Thanks, I guess I'll make the changes again

**Pull request closed without merging**